pulseapi/profiles/admin.py

```diff
@@ -42,10 +42,16 @@ class UserProfileAdmin(admin.ModelAdmin):
         'bookmark_count',
     )
 
+    def name(self, instance):
+        return instance.name or '-'
```

Comment:
given that this is the admin view only, should this be 'name or email', rather than "name or -"?

Reply:
yep, we can do that